gammapy/image/utils.py

```diff
@@ -121,11 +123,20 @@ def disk_correlate(image, radius, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : array_like
+        Image to be correlated.
+
+    radius : float
+        Disk radius in pixels.
+
+    mode : {‘reflect’,’constant’,’nearest’,’mirror’, ‘wrap’}, optional
```

Comment:
I think Sphinx generates nice-looking unicode characters in the HTML output, so if you copy & paste this from the scipy docs you introduced this issue.
My editor (PyCharm) warns me when I paste non-ascii characters into an ascii file that's not explicitly declared to be unicode ... it's an awesome IDE for Python and I can recommend it!